cache.h

```diff
@@ -1314,12 +1314,14 @@ struct object_context {
 #define GET_OID_FOLLOW_SYMLINKS 0100
 #define GET_OID_RECORD_PATH     0200
 #define GET_OID_ONLY_TO_DIE    04000
+#define GET_OID_GENTLY		  010000
 
 #define GET_OID_DISAMBIGUATORS \
 	(GET_OID_COMMIT | GET_OID_COMMITTISH | \
 	GET_OID_TREE | GET_OID_TREEISH | \
 	GET_OID_BLOB)
 
+extern int get_oid_gently(const char *str, struct object_id *oid, int gently);
```

Comment:
My instinctive reaction was that the `gently` is correct in the function name, but it would be `gentle` in the parameter name.

It seems that Git's source code agrees, by 6:3 votes (compare with `git grep -c ', int gentle'` vs `git grep -c ', int gently'`).